### DISCUSSION TITLE
Ajouter des achievements

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#1397] [Nouvelles idées]
Alors oui y'a déjà des badges
Mais justement, le but serait de faire en sorte que les badges soient des preuves d'implication dans la commu et les achievements des objectifs avec des conditions précises full tryhard et skill, et un bel affichage (un peu type missions pour le style)

**Comment by Pietagorh** [comment]
Il pourrait y avoir des achievements plus marrants ou des réfs au jeu (quand même dans des situations compliquées à effectuer)

**Comment by BastLast** [comment]
Ya pas deja une suggestion pr ça ?

**Reply by Nargacaura** [reply]
C'est la première suggestion parlant d'achievements, d'après ce que j'ai pu voir de la liste complète des suggestions.
